### DISCUSSION TITLE
Improve support for nodes with many-children

### DISCUSSION THREAD

**Discussion by dennisschaaf** [outline/outline#5534] [Ideas]
## Background

We have some folders with many children, for example a general node for 'Meetings'. Because the list of children for the meetings folder became too large we started breaking it down into a grouping page page. For example 'Meetings' -> '2023' -> 'My Meeting'.

## Challenges
- Ideally I wouldn't need the grouping pages, the side-bar should just cut-off after X items and show a 'more' button to view all.
- It is difficult to move many children around. Each document has to be moved individually. You have to scroll around the side-bar and keep loosing your spot. It would be nice to have multi-select, shift-select, etc.
- When clicking on 'more' in the side-bar the child-page list should expand to a table that has functions that make it easy to handle many pages and supports sorting/filtering by some properties.

**Comment by tommoor** [comment]
Multi-select for moving would be great indeed. 

Just so you know you can also move documents from the menu or command bar, drag and drop isn't a requirement.
